app/config/phansible/phppackages.yml

```diff
@@ -34,3 +34,32 @@ phppackages:
   - php5-xdebug
   - php5-xmlrpc
   - php5-xsl
+  - php7.0-bz2
```

Comment:
I think we should split this by version and show one or others depending on version selection.